src/common/tests/lru.c

```diff
@@ -117,13 +118,15 @@ main(int argc, char **argv)
 		exit(-1);
 	}
 
-	rc = daos_lru_cache_create(atoi(argv[1]), D_HASH_FT_RWLOCK,
+	csize	 = (int)strtol(argv[1], (char **)NULL, 10);
+	num_keys = (int)strtol(argv[2], (char **)NULL, 10);
```

Comment:
From the defect description:
tainted_data_transitive: Call to function atoi with tainted argument argv[2] returns tainted data. 
Sounds to me like complains that argv[2] is "tained", not sure it's caused by atoi or the nature of argv[]?
But (int)strtol() is better than atoi(0 according to the C programmer community :-)

Reply:
Yes it does appear coverity doesn't like atoi. So Strtol is a better option.